static/usage/common.css

```diff
@@ -237,3 +237,8 @@
 
   height: 100%;
 }
+
+:root {
```

Comment:
Should this only be needed if there's a device frame?

Reply:
Not sure off the top of my head. Maybe we can leave it until it starts causing issues for us? You're right though -- if you aren't using headers/footers there will be no visible difference.